docs/private-app-create.md

```diff
@@ -0,0 +1,92 @@
+## Glossary
+
+|Term|Description|
+|------|-------------|
+|Private App|An application/service/ETL tool that is developed by a user who wants to be able to programmatically interact with the Gateway. |
+|AppID and ClientID |Tokens that are generated when creating a new Private App. Each Private App will have a unique AppID and ClientID that needs to be sent with each API request to authenticate and approve the action. |
+|Team Admin/Developer |A role that can be assigned to a user when they are part of a ‘team’ on the Gateway. Please see the documentation [here](https://www.healthdatagateway.org/about/how-to-manage-teams-on-the-gateway) on roles and permissions on the Gateway. |
+|Authenticated API Endpoint |While a lot of the Gateway APIs are accessible without authentication, any endpoint changing data requires authentication. The ClientID acts as an authentication token. |
+
+## Introduction
+
+The Private Apps feature enables developers to create a Private App and corresponding ClientID which is used as a token to authenticate requests made to the Gateway API. This guide offers step-by-step instructions for setting up and managing Private Apps on the Gateway. 
+
+## How to Create a new Private App
+
+_Please note that HDR UK cannot create Private Apps on behalf of users._
+
+### Step 1: Sign into the Gateway
+
+Sign in to the Gateway with your preferred login provider. Make sure you have a ‘team’ set up on the Gateway. If you are not a member of an existing ‘team’ you may need to ask a Team Admin to add you  to the team (see documentation [here](https://www.healthdatagateway.org/about/how-to-manage-teams-on-the-gateway) on how to manage ‘team’ on the Gateway). If you need assistance with this step, contact the HDR UK Technology Team using the link below. 
+
+[https://www.healthdatagateway.org/about/contact-us](https://www.healthdatagateway.org/about/contact-us)
+
+
+### Step 2: Access the Gateway Private App management page
+
+Providing you have the necessary roles (Team Admin or Developer), you can access the feature by following these steps: 
+
+- Navigate to your Team Management page by clicking on your name in the top-right corner of the window, and selecting the your ’team’ from the dropdown list. 
+- In the menu on the left-hand side, select Integrations > Private Apps. 
+-   Click on the Create Private App button to initiate the process (Fig 1) 
+
+![](https://github.com/HDRUK/gateway-2-integrations-testing/assets/69473770/4dd0dbc8-10ae-46a0-b567-9c98c3367987)
+_Fig 1: How to create a new Private App_
+
+### Step 3: Enter Private App details
+
+To create a new Private App you will need to enter the following information (Fig 2):
+
+- Public App name: Name of the App you wish to create. 
+- Description: A brief description of the App. 
+- Notification contacts: Gateway user accounts for relevant notification recipients. 
+
```

Comment:
Missing text: After completing the required fields, click the Save & Continue button to proceed to the permissions page.

Reply:
Added, in next merge